server/backend_api/process/views.py

```diff
@@ -1,16 +1,36 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
 from django.http import HttpResponse
+from django.views.decorators.csrf import csrf_exempt
 from django.template import loader
 from django.core.files.storage import FileSystemStorage
 from asgiref.sync import async_to_sync
 from channels.layers import get_channel_layer
 import string
 import random
+from process.models import User
 
+
+@csrf_exempt
 def home(request):
+    username = request.POST.get('username')
+    password = request.POST.get('password')
+    for user in User.objects.all():
+        if user.username == username and user.password == password:
+            return render('load_file.html')
```

Comment:
Tu musisz do `render` jako argument przesłać jeszcze obiekt `request`

Reply:
W tym miejscu też jest problem taki, że zamiast przekierować przeglądarkę do faktycznej strony głównej, renderujesz tylko jej widok (plik `load_file.html`). W przypadku gdybyśmy dodali do strony głównej jakąś dodatkową logikę, to cała strona się posypie. Ja bym tu dał coś w stylu 
```
redirect('/')
```